source/geometry/FloatBox2D.ooc

```diff
@@ -40,7 +40,7 @@ FloatBox2D: cover {
 	rightCenter ::= FloatPoint2D new(this right, this center y)
 	topCenter ::= FloatPoint2D new(this center x, this top)
 	bottomCenter ::= FloatPoint2D new(this center x, this bottom)
-	empty ::= this size empty
+	empty ::= this size x <= 0.0f || this size y <= 0.0f
```

Comment:
We haven't defined that yet... unfortunately... 
Can we solve one problem after the other?